code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -44,7 +44,9 @@
 	if(user.faction == team)
 		user << "You can't move your own flag!"
 		return
-
+	if(loc == user)
+		if(!user.unEquip(src))
```

Comment:
Okay just be aware unEquip updates icon states and puts it on the ground and shit.